poc/mechanism/binary_randomized_response.py

```diff
@@ -0,0 +1,39 @@
+"""
+Binary randomized response mechanism based on Appendix C.1 in
+https://arxiv.org/pdf/2211.10082.pdf.
+"""
+
+import math
+
+from mechanism import DpMechanism
+
+
+class BinaryRandomizedResponse(DpMechanism):
+    DataType = list[int]
+    # Debiasing produces an array of floats.
+    DebiasedDataType = list[float]
```

Comment:
`float` is fine for now, but this raises a question I think we should surface as a GH issue: should the spec use floats or fixed precision? Relatedly, what about representing constants like `EPSILON` or `DELTA` as ratoinals? This is what we ended up doing in liprio-rs, in anticipation of this question.